src/en/charms-ha.md

```diff
@@ -56,19 +56,18 @@ overall.  Exposed ports are automatically load-balanced across all units within
 the application.  Using availability sets disables manual placement and the
 "add-machine" command.
 
-New Azure environments use availability sets by default. To disable
-this behaviour, the 'availability-sets-enabled' option must be set in
-environments.yaml like so:
+New Azure environments use availability sets by default. This behaviour can be
+disabled only when bootstrapping the cloud by adding
+'availability-sets-enabled=false' as a configuration option:
 
 ```yaml
```

Comment:
Did you want to continue to format this as 'yaml' or instead change it to 'bash', like most other text boxes?